designer/server/src/routes/forms/create.js

```diff
@@ -316,3 +320,7 @@ function redirectToTitleWithErrors(request, h) {
  * @typedef {import('@hapi/hapi').ResponseToolkit<any>} ResponseToolkit
  * @typedef {Request<{ Payload: any }>} RequestWithPayload
  */
+
+/**
+ * @typedef {import('@hapi/hapi').RouteOptionsValidate['failAction']} FailAction
```

Comment:
The `failAction` property is already typed if you want to use `Lifecycle.FailAction` or `FailAction`?

```suggestion
 * @typedef {import('@hapi/hapi').Lifecycle} Lifecycle
```

```suggestion
 * @typedef {import('@hapi/hapi').Lifecycle.FailAction} FailAction
```

Reply:
Thanks! I've removed it, I actually I wasn't even using this type, it was leftover code from a previous iteration.